presto-main/src/main/java/com/facebook/presto/server/JavaVersion.java

```diff
@@ -27,8 +27,8 @@
 // TODO: remove this when we upgrade to Java 9 (replace with java.lang.Runtime.getVersion())
```

Comment:
Is this TODO still relevant?

Reply:
Yes, we could use java.lang.Runtime.getVersion() instead of the version parser once we upgrade the minimum java version from 8 to 9.